gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -531,6 +531,8 @@
    * Configuration properties used by the CopySource.
    */
   public static final String COPY_SOURCE_FILESET_WU_GENERATOR_CLASS = "copy.source.fileset.wu.generator.class";
+  public static final String GOBBLIN_COPY_PREFIX = "gobblin.copy";
+  public static final String GOBBLIN_COPY_WORK_UNIT_WEIGHT = GOBBLIN_COPY_PREFIX + ".workUnitWeight";
```

Comment:
Why should this config be in ConfigurationKeys as opposed to CopySource/CopyConfiguration?

Reply:
Oh, I think this was a remnant from an attempt to implement the GobblinSplit.getLength method  before I realized we already do work unit bin packing so this wouldn't be something that would be very useful - I guess I forget to change this back to the way it was before...